feg/gateway/Vagrantfile

```diff
@@ -84,7 +82,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     feg_prod_bare.vbguest.auto_update = false
     feg_prod_bare.vm.network "private_network", ip: "192.168.91.10"
     feg_prod_bare.vm.network "private_network", ip: "192.168.81.100"
-    config.vm.provider "virtualbox" do |v|
+    feg_prod_bare.vm.provider "virtualbox" do |v|
       v.customize ['modifyvm', :id, '--natnet1', '10.0.3.0/24']
       v.linked_clone = true
       v.memory = 512
```

Comment:
changes on this Vagrant file are just in case we ever need to use a VM to build/run/test feg. This modified VM supports those three functions